src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingUtils.kt

```diff
@@ -485,7 +491,10 @@ fun Vilkårsvurdering.oppdaterMedDødsdatoer(personopplysningGrunnlag: Personopp
     }
 }
 
-fun Vilkårsvurdering.kopierResultaterFraForrigeBehandling(vilkårsvurderingForrigeBehandling: Vilkårsvurdering) {
+fun Vilkårsvurdering.kopierResultaterFraForrigeBehandling(
+    vilkårsvurderingForrigeBehandling: Vilkårsvurdering,
+    behandlingSkalFølgeNyeLovendringer2024: Boolean,
```

Comment:
Vet at navnet ikke kommer fra deg, men fortrekker at boolean het skalBehandlingFølgeNyeLovendringer2024, så den holder seg på mer standard navnekonvesjon for boolske verdier

Reply:
Fiksa i [Rename behandlingSkalFølgeNyeLovendringer2024 -> erToggleForLovendrin…](https://github.com/navikt/familie-ks-sak/pull/726/commits/922f5ebdb4487adcf3ffff5c5ac4d8d61bd15c56)